lib/server.js

```diff
@@ -123,9 +123,9 @@ module.exports = (config) => {
             // Initialize common data in buildFactory and jobFactory
             if (server.app.buildFactory) {
                 server.app.buildFactory.apiUri = server.info.uri;
-                server.app.buildFactory.tokenGen = (buildId, metadata, scmContext) =>
+                server.app.buildFactory.tokenGen = (buildId, metadata, scmContext, expiresSec) =>
```

Comment:
`tokenGen()` method is called in [models](https://github.com/screwdriver-cd/models/pull/245/files#diff-26b681bd4dac10f25d5293c7595c435bR170)